src/mps/dmrg.jl

```diff
@@ -316,7 +316,7 @@ function dmrg(PH, psi0::MPS, sweeps::Sweeps; kwargs...)::Tuple{Number,MPS}
   return (energy, psi)
 end
 
-function _dmrg_sweeps(; nsweeps, maxdim, mindim=1, cutoff=1e-8, noise=0.0, kwargs...)
+function _dmrg_sweeps(; nsweeps, maxdim=10_000, mindim=1, cutoff=1E-8, noise=0.0, kwargs...)
```

Comment:
What if we just use `typemax(Int)` (which equals `9223372036854775807`)? I'd prefer that over choosing an arbitrary cutoff, it seems either way the calculation would get very slow or run out of memory and the user would find out quickly they did something wrong.

Reply:
Sounds good – just made that change